packages/flutter/test/widgets/tap_region_test.dart

```diff
@@ -253,7 +338,7 @@ void main() {
         tester.getCenter(finder),
         kind: PointerDeviceKind.mouse,
       );
-      await gesture.up();
+      // We intentionally don't call up() here because we're testing the down event.
```

Comment:
@polina-c is there a memory leak if a gesture doesn't have `.up` called? Do we need to use `.cancel` here?